executers/dependenciesutils.go

```diff
@@ -154,17 +154,35 @@ func createDependencyInTemp(zipPath string) (tempDir string, err error) {
 	return tempDir, nil
 }
 
-func replaceExclamationMarkWithUpperCase(moduleName string) string {
+// Returns the actual path to the dependency.
+// If in the path there are capital letters, the Go convention is to use "!" before the letter.
+// The letter itself in lowercase.
+func goModEncode(name string) string {
+	path := ""
+	for _, letter := range name {
+		if unicode.IsUpper(letter) {
+			path += "!" + strings.ToLower(string(letter))
+		} else {
+			path += string(letter)
+		}
+	}
+	return path
+}
+
+// Returns the value with upper case.
```

Comment:
The above comment line is incorrect grammatically.

Reply:
Changed it.